news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – July 2020 edition"
+subtitle: "Where you'll learn that that there is a way to boost translations"
+date:  2020-08-07 10:00:00
+authors: [JuneByun]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in April.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+
+### First news, a way to boost translations :fire:
+
+As I mentioned in the [previous Crowdin Monthly](https://build.prestashop.com/news/do-you-speak-prestashop-june-2020/), many PrestaShop translation projects, such as Esperanto, are reaching higher translation rates thanks to dedicated contributors. To help and encourage them, I did some tests by translating strings with Crowdin’s Pre-translation via Translation Memory (TM) tool. What is this tool and how does it work? 
+
+The Project Translation Memory (TM) is the vault of the source strings and their translations into different languages. Every translation made in a project is automatically added to the Project Translation Memory. By using the Pre-translation via the TM tool, translations of the same or similar strings in a project can be applied to untranslated strings. As strings of older versions are propagated to new versions, the more the older versions are translated, the less you will have to translate for new versions. :sparkles:
+
+
+### Interested in testing the TM tool? 
+
+This test was done for the Spanish, Mexico and Spanish, Colombia project and the translation rate went respectively from 70% to 96% and 65% to 76%. :tada: To conclude this test, proofreaders of these projects are needed to vouch for the translation consistency and approve these strings. Any latin american spanish native who wishes to help and become more involved in projects by becoming proofreaders are more than welcome. 
+
+I also want to let contributors know that if you are interested in testing the TM tool to translate more strings of the language project you are working on, you can [reach out to me](https://crowdin.com/profile/june.byun)! If translations done by the TM tool are proved to be accurate, this method could be replicated and make your work easier and make translations available for merchants - and you as well. :wink:
```

Comment:
It's interesting, you should consider writing a dedicated blog post instead. ;-)

Reply:
```suggestion
### First news, a way to boost translations :fire:

As I mentioned in the [previous Crowdin Monthly](https://build.prestashop.com/news/do-you-speak-prestashop-june-2020/), many PrestaShop translation projects, such as Esperanto, are reaching higher translation rates thanks to dedicated contributors. To help and encourage them, I did some tests by translating strings with Crowdin’s Pre-translation via Translation Memory (TM) tool. What is this tool and how does it work? 

The Project Translation Memory (TM) is the vault of the source strings and their translations into different languages. Every translation made in a project is automatically added to the Project Translation Memory. By using the Pre-translation via the TM tool, translations of the same or similar strings in a project can be applied to untranslated strings. As strings of older versions are propagated to new versions, the more the older versions are translated, the less you will have to translate for new versions. :sparkles:


### Interested in testing the TM tool? 

This test was done for both the Spanish, Mexico and Spanish, Colombia project and the translation rate went respectively from 70% to 96% and 65% to 76%. :tada: To conclude this test, proofreaders of these projects are needed to vouch for the translation consistency and approve these strings. Any Latin American Spanish native who wishes to help and become more involved in projects by becoming proofreaders are more than welcome. 
```

Reply:
I'm not in favor of the last paragraph: in my opinion, the TM tool should remain an exception in the translation process, and definitely not be an option for any contributor.